README.md

```diff
@@ -125,18 +139,27 @@ may need to add `~/go/bin` to your \$PATH (MacOS/Linux), or `%HOME%\go\bin`
 (Windows). Not to be mistaked for `C:\Go\bin` (which is for Go's own binaries,
 not apps like Lazygit).
 
+### Binary Releases
```

Comment:
I think binary releases was requested to go to the top of this list from someone because it was a common thing that people missed, so I would chuck it back at the top